pkg/boot/linux/load_linux_x86.go

```diff
@@ -2,14 +2,16 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
+//go:build linux && amd64
```

Comment:
note you don't need the build tags; the file name does the trick -- if you use the right name.
This name should be load_linux_amd64.go

Reply:
Cool beans !

Need match goarch